packages/mobile/src/exchange/saga.ts

```diff
@@ -243,7 +258,7 @@ export function* exchangeGoldAndStableTokens(action: ExchangeTokensAction) {
     const exchangeContract: ExchangeWrapper = yield call(getExchangeContract, stableToken)
 
     const convertedMakerAmount: BigNumber = roundDown(
-      yield call(convertToContractDecimals, makerAmount, makerToken),
+      yield call(convertToContractDecimals, inputAmount, makerToken),
```

Comment:
Why this change? The `makerAmount` is the amount of the token we'll sell, and the `inputAmount` is in the input which could be CELO or local currency, right?

Reply:
Yes, but I used `inputAmount` because the maker amount comes in the incorrect token, I mean:
- Buying Celo the `makerAmount` is:
  - Using Celo as input: you buy 1 and you get 1.
  - Using stable as input: you buy 1 and you get the conversion of 1. So it is less accurate.